src/components/Calculator.js

```diff
@@ -0,0 +1,37 @@
+import './calculator.css';
+/* eslint-disable import/no-extraneous-dependencies */
+import PropTypes from 'prop-types';
+
+function Calculator(props) {
+  const { onClick } = props;
+
+  return (
+    <div className="container">
+      <button type="button" onClick={onClick} className="btn">AC</button>
+      <button type="button" onClick={onClick} className="btn">+/-</button>
+      <button type="button" onClick={onClick} className="btn">%</button>
+      <button type="button" onClick={onClick} className="btn divide">÷</button>
+      <button type="button" onClick={onClick} className="btn">7</button>
+      <button type="button" onClick={onClick} className="btn">8</button>
+      <button type="button" onClick={onClick} className="btn">9</button>
+      <button type="button" onClick={onClick} className="btn multiply">×</button>
```

Comment:
- [ ] Please update your design to match the example given in the project requirements 😉 

-Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/220712534-4d69a471-61ac-4290-9f2f-86954ed0ab0b.png)

- [ ] Original design 👇 

![image](https://user-images.githubusercontent.com/75126481/220712850-dd1b23bc-e51a-452d-b428-9803aff7c35b.png)